examples/image-points-3d.py

```diff
@@ -14,9 +14,11 @@
 nuclei = cells[:, 1]
 smooth = filters.gaussian(nuclei, sigma=10)
 pts = feature.peak_local_max(smooth)
-viewer = napari.view_image(
-        cells, channel_axis=1, name=['membranes', 'nuclei'], ndisplay=3
+viewer = napari.Viewer()
+layer = viewer.add_image(
+        cells, channel_axis=1, name=['membranes', 'nuclei']
         )
```

Comment:
```suggestion
membranes, nuclei = viewer.add_image(
    cells, channel_axis=1, name=['membranes', 'nuclei']
)
```
So, this is the great thing about `add_image` in that it returns the layers, which we didn't have before. This still passed the tests because its technically ok, but in reality, `add_image` will return a tuple of the length of the number of layers added. So in this case, we get two layers, one named membranes and one named nuclei. You could call these `layer0, layer1` or `membranes, nuclei` in this case, it doesn't matter because the notebook doesn't refer to it, but it'd be good for us to model the behavior in the example!

Reply:
thanks @TimMonko , for mentioning this . Going to revisit the code and correct my mistakes